0001-short-term-release-process/0001-release-process-short-term.md

```diff
@@ -0,0 +1,325 @@
+# Short-Term Release Process for Autonomi
+
+- Status: proposed
+- Type: enhancement
+- Related components:
+- Start Date: 2024-07-08
+- Discussion:
+- Supersedes:
+- Superseded by:
+
+## Summary
+
+This document intends to provide a detailed development and release process we can follow until the
+initial launch of the network in October/November. It will be based on short release cycles that
+will last about three weeks. After launch, the cycles would likely be at least double that length
+and could be quite different in detail.
+
+It should be clear that we deal here with processes for releasing code, but we don't go into any
+detail on how to deploy it. There are issues there we will also have to address, but those will be
+for another document.
+
+## Conventions
+
+Uncertainties are indicated using a `*` notation. Rather than have a full `Unresolved Questions`
+section at the end, where applicable there is an `Uncertainties` list in each section.
+
+The use of `**` is to denote an idea that came from Shu's diagram/proposal.
+
+## Motivations
+
+So far, we have failed to arrive at a well-defined process for releasing our code. With launch
+imminent, it is my opinion that we've now run out of road for experimentation and we need to settle
+on an unambiguous process that we can execute without deliberation. Finalising this document should
+be a collaborative process in which we address any uncertainties rather than leaving them open for
+further experiment.
+
+## Detailed Design
+
+## Beta Release Cycles and Schedule
+
+This section provides guidelines for possible dates for each cycle until launch.
+
+### Wave 1
+
+Release and deployment for next wave:
+* Stable release and production deploy: 2024-07-08
+
+### Wave 2
+
+Live for users: 2024-07-09 / 825 people / 11 week comp / comp ends: 2024-09-27
```

Comment:
> comp ends: 2024-09-27

what does `comp ends` mean here?

Reply:
Ah sorry, the schedule here was actually based on a Slack post from Bux, and she put this phrase in each of the phases. I think it means "competition ends". I will clarify that with her and update.